src/SFA.DAS.AssessorService.Web/Views/Certificate/Check.cshtml

```diff
@@ -41,6 +42,7 @@ else
                     <div class="govuk-error-summary__body">
                         <ul class="govuk-list govuk-error-summary__list">
                             <li><a sfa-validation-for="Name"></a></li>
+                            <li><a sfa-validation-for="Option"></a></li>
```

Comment:
Option should be higher up the list - the otehr fields all relate to the postal address details